src/term/net_to_term.rs

```diff
@@ -207,6 +210,25 @@ impl<'a> Reader<'a> {
   }
 }
 
+fn decode_str(fst: Term, bod: Term) -> Term {
```

Comment:
These `unreachable!`s should instead push readback errors (as many of them are likely reachable in debug mode)